.gitignore

```diff
@@ -0,0 +1,19 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
+/dist
+/tmp
+
+# dependencies
+/node_modules
+/bower_components
+
+# misc
+/.sass-cache
+/connect.lock
+/coverage/*
```

Comment:
I'm not sure this will work once we get coverage actually working. At least, I had to keep a `coverage/.gitkeep` so that the coverage directory was there for the `lcov.info` to be placed in during the test run. Seemed like `ember-cli-blanket` wouldn't create the `coverage/` directory if it wasn't there.